### PR TITLE
[New Feature]Added internationalization configuration to Fury.

### PR DESCRIPTION
**Pull request by LofiSu**
"Added internationalized routing configuration and internationalization framework setup to Fury, and included a Simplified Chinese version."
image
image

### CODE REVIEW

File: i18n/en/code.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
+{
+  "theme.ErrorPageContent.title": {
+    "message": "This page crashed.",
```
Comment (chaokunyang) [inline i18n/en/code.json:3]: Whylen will we go to this page

File: i18n/en/code.json [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    "description": "The title of the fallback page when the page crashed"
+  },
+  "theme.BackToTopButton.buttonAriaLabel": {
+    "message": "Scroll back to top",
+    "description": "The ARIA label for the back to top button"
+  },
+  "theme.blog.archive.title": {
+    "message": "Archive",
```
Comment (chaokunyang) [inline i18n/en/code.json:11]: What do you mean archive page

**Comment by chaokunyang** [comment]
@theweipeng  could you take a look at this pr?

**Review by LofiSu — COMMENTED** [review]
The unnecessary parts have been deleted. Some come with the npm run start -- --locale zh-CN command integration. It does not affect the running, just translates.

File: i18n/en/code.json (hunk 2 above)
Comment (LofiSu) [reply i18n/en/code.json]: The two you mentioned can be deleted directly. I used the command npm run write-translations -- --locale zh-Hans, which will automatically integrate the code.js file into i8n. If the page does not have it, you can delete it.

File: i18n/zh-CN/code.json [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+  "theme.blog.paginator.olderEntries": {
+    "message": "较旧的博文",
+    "description": "The label used to navigate to the older blog posts page (next page)"
+  },
+  "theme.blog.post.paginator.navAriaLabel": {
+    "message": "博文分页导航",
+    "description": "The ARIA label for the blog posts pagination"
+  },
+  "theme.blog.post.paginator.newerPost": {
+    "message": "较新一篇",
```
Comment (chaokunyang) [inline i18n/zh-CN/code.json:23]: 下一篇

File: i18n/zh-CN/code.json [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+    "description": "The blog post button label to navigate to the newer/previous post"
+  },
+  "theme.blog.post.paginator.olderPost": {
+    "message": "较旧一篇",
```
Comment (chaokunyang) [inline i18n/zh-CN/code.json:27]: 上一篇

**Review by chaokunyang — APPROVED** [review]
LGTM

**Pull request merged**